docker/docker_client.go

```diff
@@ -312,6 +339,7 @@ func (c *dockerClient) makeRequestToResolvedURL(ctx context.Context, method, url
 //
 // debugging: https://github.com/containers/image/pull/211#issuecomment-273426236 and follows up
 func (c *dockerClient) setupRequestAuth(req *http.Request) error {
+	var scope string
```

Comment:
(Non-blocking: Consider moving the `scope` declaration just before its first use; it’s used only in `case "bearer"`.)

Reply:
fixed